app/src/main/java/com/bogdan/codeforceswatcher/features/users/UsersAdapter.kt

```diff
@@ -87,7 +87,7 @@ class UsersAdapter(
 
         init {
             view.setOnClickListener {
-                itemClickListener.invoke(adapterPosition)
+                if (adapterPosition != RecyclerView.NO_POSITION) itemClickListener.invoke(adapterPosition)
```

Comment:
Check https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.ViewHolder.html#getAdapterPosition()
One user has a crash connected to it.